plugin/src/PatchTree.lua

```diff
@@ -16,6 +16,14 @@ local Types = require(Plugin.Types)
 local decodeValue = require(Plugin.Reconciler.decodeValue)
 local getProperty = require(Plugin.Reconciler.getProperty)
 
+local function yieldIfNeeded(clock)
+	if os.clock() - clock > 1 / 20 then
```

Comment:
1/20 of a second feels like an arbitrary time. `task.wait()` is something like 1/30th of a second, right?